### PR TITLE
Create acr-wcag-21-sc.json-ld

### PR DESCRIPTION
**Pull request by BrahimMahadi**
Added the WCAG 2.0 SC

### CODE REVIEW

File: context/2022/acr-wcag-21-sc.json-ld [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+	"WCAG21:animation-from-interactions": { "@type": "earl:TestRequirement" },
+	"WCAG21:target-size": { "@type": "earl:TestRequirement" },
+	"WCAG21:concurrent-input-mechanisms": { "@type": "earl:TestRequirement" },
+	"WCAG22:accessible-authentication-no-exception": { "@type": "earl:TestRequirement" },
+	"WCAG22:focus-appearance-enhanced": { "@type": "earl:TestRequirement" },
+	
+	"WCAG21Tech":"https://www.w3.org/WAI/WCAG21/Techniques/",
+	"WCAG22Tech":"https://www.w3.org/WAI/WCAG22/Techniques/",
+	
+  "WCAG21Tech:html/H44": { "@type": "earl:TestCase" },
```
Comment (duboisp) [inline context/2022/acr-wcag-21-sc.json-ld:99]: There is some addtional techniques. Voici tout les techniques: https://www.w3.org/WAI/WCAG22/Techniques/

File: context/2022/acr-wcag-21-sc.json-ld [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+	"techniques": { "@id": "earl:test", "@type": "@id" },
+	"advisory": { "@id": "earl:test", "@type": "@id" },
+	"outcome": { "@id": "wb-acr:outcome", "@type": "xsd:string" },
+
+	"info": { "@id": "earl:info" },
+	"label": { "@id": "dct:title" },
+	"notes": { "@id": "dcterms:description" },
+	"date": { "@id": "dcterms:date" },
+	
+	"earl:mode": { "@type": "@id" }
```
Comment (duboisp) [inline context/2022/acr-wcag-21-sc.json-ld]: Add alias for the:
* wb-acr:severity
* wb-acr:relevancy

Plus its corresponding status

* wb-acr:critical
* wb-acr:moderate

**Review by duboisp — CHANGES_REQUESTED** [review]

File: context/2022/acr-wcag-21-sc.json-ld (hunk 1 above)
Comment (BrahimMahadi) [reply context/2022/acr-wcag-21-sc.json-ld:99]: Added additional techniques

File: context/2022/acr-wcag-21-sc.json-ld (hunk 2 above)
Comment (BrahimMahadi) [reply context/2022/acr-wcag-21-sc.json-ld]: Added alias for both severity and relevancy in recent commit

File: context/2022/acr-wcag-21-sc.json-ld [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
```
Comment (duboisp) [inline context/2022/acr-wcag-21-sc.json-ld]: JSON parsing error: Missing a comma at the end,

**Review by duboisp — CHANGES_REQUESTED** [review]

File: context/2022/acr-wcag-21-sc.json-ld (hunk 3 above)
Comment (duboisp) [inline context/2022/acr-wcag-21-sc.json-ld]:
```suggestion
	"earl:mode": { "@type": "@id" },
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Pull request merged**